test/libsolidity/smtCheckerTests/special/timestamp_2.sol

```diff
@@ -0,0 +1,9 @@
+pragma experimental SMTChecker;
+
+contract C {
+	uint x;
+
+	constructor() {
+		x = block.timestamp + 0; // No overflow should be reported
```

Comment:
Could you add a test for `block.timestamp + 1` if it does not exist already?

Reply:
Added.